Build failed in example project

Steps to reproduce:

- Clone the project
- Open Console in ''games_services\games_services\example"
- execute 'flutter run'

Result:
```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:processDebugMainManifest'.
> Incorrect package="com.abedalkareem.games_services_example" found in source AndroidManifest.xml: D:\work\github\games_services\games_services\example\android\app\src\main\AndroidManifest.xml.
  Setting the namespace via the package attribute in the source AndroidManifest.xml is no longer supported.
  Recommendation: remove package="com.abedalkareem.games_services_example" from the source AndroidManifest.xml: D:\work\github\games_services\games_services\example\android\app\src\main\AndroidManifest.xml.
```

--- a/games_services/example/android/app/src/main/AndroidManifest.xml
+++ b/games_services/example/android/app/src/main/AndroidManifest.xml
@@ -1,5 +1,4 @@
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.abedalkareem.games_services_example">
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
 
     <!-- io.flutter.app.FlutterApplication is an android.app.Application that
          calls FlutterMain.startInitialization(this); in its onCreate method.

